api/management-model/build.gradle.kts

```diff
@@ -29,6 +29,9 @@ dependencies {
   compileOnly(libs.jakarta.annotation.api)
   compileOnly(libs.jakarta.validation.api)
   compileOnly(libs.swagger.annotations)
+  testImplementation(platform("org.junit:junit-bom:5.9.2"))
```

Comment:
Please use `testImplementation(platform(libs.junit.bom))` - the version is defined in `gradle/libs.versions.toml`

Reply:
nit: maybe add a new line above?